src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx

```diff
@@ -118,7 +54,7 @@ const AdvancedSettings: React.FC<AdvancedSettingsProps> = ({
 
   const values = watch();
 
-  const onSubmit = (data: Venue_v2_AdvancedConfig) => {
+  const updateAdvancedSettings = (data: VenueAdvancedConfig) => {
```

Comment:
Can I ask a minor adjustment here so we actually capture possible errors, even if just in console?

```tsx
    updateVenue_v2(
      {
        name: venue.name,
        worldId: venue.worldId,
        ...data,
      },
      user
    ).catch((e) => console.error(AdvancedSettings.name, e));
```

Reply:
The columns are now fixed and console.error has been added.

Reply:
@denisdimitrov 
> How about Bugsnag?
Nothing against it. As long as errors aren't ignored. Any particular context we want to add to Admin related errors?